staging/src/k8s.io/kubectl/pkg/cmd/create/create_configmap.go

```diff
@@ -285,6 +289,19 @@ func (o *ConfigMapOptions) createConfigMap() (*corev1.ConfigMap, error) {
 		configMap.Name = fmt.Sprintf("%s-%s", configMap.Name, hash)
 	}
 
+	fmt.Println("11111111111111!!!")
+	if len(o.Labels) > 0 {
+		fmt.Println("----------11111111111111!!!")
+		labels, err := cmdutil.ParseLabels(o.Labels)
+		if err != nil {
+			fmt.Printf("EEEEEEEEEEEEEE: %v", err)
+			return nil, err
+		}
+		configMap.Labels = labels
+
+		fmt.Printf("..........11111111111111!!!:  %v\n", configMap.Labels)
+	}
+
```

Comment:
I would say you are still developing and debugging, right? :D

Reply:
yes @rikatz definitely debugging. I already got a new commit that removes those lines and adds unit testing.
I have a question, given that this PR is for configmaps but we agreed that this global flag can be used for other resources, should I create different PRs for each resource?